.pipelines/.vsts-vhd-builder.yaml

```diff
@@ -59,6 +59,7 @@ phases:
           -e SIG_GALLERY_NAME=${SIG_GALLERY_NAME} \
           -e SIG_IMAGE_NAME=${SIG_IMAGE_NAME} \
           -e SIG_IMAGE_VERSION=${SIG_IMAGE_VERSION} \
+          -e GEN2_SGX_DRIVER_INSTALL=$(GEN2_SGX_DRIVER_INSTALL) \
```

Comment:
not necessary

Reply:
yes since it is hardcoded no need for this one